api/rooms/views/main.py

```diff
@@ -0,0 +1,38 @@
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
+
+from ..models import Room
+
+class RoomSuggestionView(View):
+    def get(self, request):
+        try: 
+            keyword_sort = request.GET.get('keyword_sort','서울')
+            offset       = int(request.GET.get('offset', 0))
+            limit        = int(request.GET.get('limit', 1))
+            
+            rooms = Room.objects.select_related('location','room_type','trade_type')\
+                    .prefetch_related('roomimage_set')\
+                    .filter(Q(location__dong__icontains=keyword_sort) | Q(location__city__icontains=keyword_sort))\
```

Comment:
dong과 city를 서울로 필터링 하는 건가요?
```suggestion
                    .filter(Q(location__dong__icontains=keyword_sort) | Q(location__city__icontains=keyword_sort))\
```

Reply:
서울 붙인건 ... default인줄 알았는데...좀 더 알아보겠습니다~

Reply:
아란님 맞아요!! .get('key_word', default) 호출했을 때, 해당 key 값이 없으면 2번째 인자로 넣어준 값이 default로 반환됩니다!
제가 궁굼했던 건 filter를 dong, city를 둘다 서울로 하는 것 같아서 여쭤봤어요!!

`dong` 데이터에도 `서울`이라는 키워드가 포함되어 있는 건가요?

Reply:
아니요 dong에는 동만... 기준이 필요하겠군요 !!! 무슨말씀인지 알거같아요!